.changeset/bright-maps-jump.md

```diff
@@ -0,0 +1,6 @@
+---
+"@comet/cms-admin": minor
+"@comet/cms-api": minor
+---
+
+Add support for external dams
```

Comment:
```suggestion
Add support for external DAMs
```

Please be more specific here: What do I need to do? Maybe add a note about the example in Demo

Reply:
> Please be more specific here: What do I need to do? Maybe add a note about the example in Demo

This still needs to be done

Reply:
Is the new changeset sufficient?